### PR TITLE
Update account page - personal data

### PR DESCRIPTION
**Pull request by AndreeaSburlea**
## New features
- show the personal data on account page

## Changes in code
- update the personal data from google account
- update the cart

## Screenshot
Screenshot 2022-06-24 at 14 30 23

### CODE REVIEW

File: ShopApp/ShopApp/Controller/PersonalDataViewController.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        // set photoImageView properties
+        photoImageView.layer.borderWidth = 1.0
+        photoImageView.layer.masksToBounds = false
+        photoImageView.layer.borderColor = UIColor.white.cgColor
+        photoImageView.layer.cornerRadius = photoImageView.frame.size.width / 2
+        photoImageView.clipsToBounds = true
+
+        getEmail()
+
+    }
```
Comment (VivianaSara) [inline ShopApp/ShopApp/Controller/PersonalDataViewController.swift]: delete empty line

File: ShopApp/ShopApp/Controller/PersonalDataViewController.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    func getEmail() {
+        guard let emailAddress = Auth.auth().currentUser?.email as? String else { return }
+        guard let name = Auth.auth().currentUser?.displayName as? String else { return }
```
Comment (VivianaSara) [inline ShopApp/ShopApp/Controller/PersonalDataViewController.swift]: maybe this function should has a more specific name like "getUseInfo"

File: ShopApp/ShopApp/Controller/PersonalDataViewController.swift [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        guard let imageUrl = Auth.auth().currentUser?.photoURL as? URL else { return }
+
+        downloadImage(from: imageUrl)
+        emailLabel.text = emailAddress
+        nameLabel.text = name
+
+    }
```
Comment (VivianaSara) [inline ShopApp/ShopApp/Controller/PersonalDataViewController.swift]: delete empty line

File: ShopApp/ShopApp/Controller/PersonalDataViewController.swift [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        getData(from: url) { data, response, error in
+            guard let data = data, error == nil else { return }
+            // always update the UI from the main thread
+            DispatchQueue.main.async() { [weak self] in
+                self?.photoImageView.image = UIImage(data: data)
+            }
+        }
+    }
+
+}
```
Comment (VivianaSara) [inline ShopApp/ShopApp/Controller/PersonalDataViewController.swift]: delete empty line

File: ShopApp/ShopApp/Controller/PersonalDataViewController.swift (hunk 1 above)
Comment (VivianaSara) [inline ShopApp/ShopApp/Controller/PersonalDataViewController.swift]: delete empty line

File: ShopApp/ShopApp/Controller/PersonalDataViewController.swift [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    @IBOutlet private var photoImageView: UIImageView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // set photoImageView properties
+        photoImageView.layer.borderWidth = 1.0
+        photoImageView.layer.masksToBounds = false
+        photoImageView.layer.borderColor = UIColor.white.cgColor
+        photoImageView.layer.cornerRadius = photoImageView.frame.size.width / 2
+        photoImageView.clipsToBounds = true
```
Comment (VivianaSara) [inline ShopApp/ShopApp/Controller/PersonalDataViewController.swift:30]: you can create a function of these configurations

**Review by VivianaSara — APPROVED** [review]

File: ShopApp/ShopApp/Controller/CartTableViewController.swift [hunk 6]
```diff
@@ -157,7 +166,6 @@ class CartTableViewController: UITableViewController {
                     return
                 }
             }
-
```
Comment (paulpop10) [inline ShopApp/ShopApp/Controller/CartTableViewController.swift:160]: This empty line should have stayed

File: ShopApp/ShopApp/Controller/PersonalDataViewController.swift [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    @IBOutlet private var emailLabel: UILabel!
+    @IBOutlet private var photoImageView: UIImageView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        photoProperties()
+        getUserInfo()
+    }
+
+    func photoProperties() {
```
Comment (paulpop10) [inline ShopApp/ShopApp/Controller/PersonalDataViewController.swift]: All method names should contain a verb. This one can be like "setPhotoProperties"

**Review by paulpop10 — APPROVED** [review]

**Pull request merged**